devel/clavus/Portfile

```diff
@@ -0,0 +1,43 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:filetype=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           rust 1.0
+
+name                clavus
+version             0.0.1
+categories          devel
+
+maintainers         {gmail.com:pietro.mobrici}
+
+platforms           darwin
+supported_archs     x86_64
+
+description         Clavum Lateris
+long_description    In vetusta bibliotheca, clavum lateris aperiebat fores librarii occultum, ubi sapientia et arcana conservabantur.
+
+homepage            https://github.com/peter-mbx/clavus
+
+github.setup        peter-mbx clavus 0.0.1
+
+checksums               sha256 bb64672b75d2d459063820b65364798ab9569e14e7ee04a1d5ce514c3c5c18b4 \
+                        size 11428
+
+distname            ${name}-${version}
+use_bzip2           no
+
+depends_build       port:cargo
+depends_lib         port:rust
+
+configure {}
+
+build {
+    system "cd ${workpath}/${distname} && cargo build --release"
+}
+
+destroot {
+    xinstall -m 755 ${workpath}/${name}-${version}/target/release/clavus ${destroot}${prefix}/bin/
+}
```

Comment:
not sure about these dependencies and overriding the `configure`, `build`, and `destroot` phase. Why are you using the `rust` PortGroup but then nothing of the settings that are used in there? Should/could this not use the the `cargo` PortGroup instead? 

To be honest, I don't know but this seems a bit unusual so I am not convinced the PortGroups are used appropriately and that should likely be investigated a bit

Reply:
Hi @reneeotten, thank you for your review.
I've marked the other conversations as resolved because I updated the Portfile based on your feedback.

Regarding this conversation, I'm relatively new to Rust, so I took a look at other ports and experimented to make it work.
Your guidance is greatly appreciated, and any further advice is always welcome.